addons/snailmail/models/snailmail_letter.py

```diff
@@ -460,25 +464,33 @@ def _append_cover_page(self, invoice_bin: bytes):
 
         cover_buf = io.BytesIO()
         canvas = Canvas(cover_buf, pagesize=A4)
+        font_path = get_module_resource('web', 'static/fonts/lato/Lato-Reg-webfont.ttf')
+        pdfmetrics.registerFont(TTFont("Lato", font_path))
```

Comment:
Is there a reason to choose this font in particular?

Reply:
Initially I thought it was the most fitting font we had included in Odoo, but upon further testing I noticed it doesn't render non-Latin alphabets, none of the other included font seem to be able to.

So we'll probably need to include new fonts in the snailmail module if we want to support those.